test/built-ins/Promise/executor-function-name.js

```diff
@@ -22,3 +23,6 @@ function NotPromise(executor) {
 Promise.resolve.call(NotPromise);
 
 assert.sameValue(Object.prototype.hasOwnProperty.call(executorFunction, "name"), false);
+assert.sameValue(Object.getOwnPropertyNames(executorFunction).join(), "length");
+assert.sameValue(delete executorFunction.name, true);
```

Comment:
why the delete here after `assert.sameValue(Object.prototype.hasOwnProperty.call(executorFunction, "name"), false);`?

Reply:
I bet the original test was written at the time when `Function.name` was non-standard and non-configurable.
Totally useless, sorry.

71e1291